<ui version="4.0" >
 <class>AudioClipEditWidget</class>
 <widget class="QWidget" name="AudioClipEditWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Clip parameters</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Gain</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Track start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="clipNameLineEdit" />
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QDoubleSpinBox" name="clipGainSpinBox" >
              <property name="maximum" >
               <double>30.000000000000000</double>
              </property>
              <property name="minimum" >
               <double>-120.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="clipLengthSpinBox" >
              <property name="decimals" >
               <number>3</number>
              </property>
              <property name="maximum" >
               <double>1200.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="clipStartSpinBox" >
              <property name="suffix" >
               <string/>
              </property>
              <property name="decimals" >
               <number>3</number>
              </property>
              <property name="maximum" >
               <double>1200.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QPushButton" name="externalProcessingButton" >
              <property name="text" >
               <string>External Processing</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Fade In</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_4" />
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Fade Out</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_5" />
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
